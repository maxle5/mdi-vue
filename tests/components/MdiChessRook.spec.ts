
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiChessRook from "../../src/components/MdiChessRook.vue";

test("MdiChessRook snapshot", () => {
  const wrapper = mount(MdiChessRook, {});
  expect(wrapper.html()).toMatchSnapshot();
});
