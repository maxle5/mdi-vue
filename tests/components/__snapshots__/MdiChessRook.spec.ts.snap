// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiChessRook snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M5,20H19V22H5V20M17,2V5H15V2H13V5H11V2H9V5H7V2H5V8H7V18H17V8H19V2H17Z" style="fill: currentColor;"></path>
</svg>"
`;
